site/content/en/docs/tasks/run/multikueue/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Using MultiKueue"
+linkTitle: "MultiKueue"
+weight: 8
+date: 2024-09-25
+description: >
+  The workloads that support MultiKueue.
+---
+
+This page explains how to run tasks in MultiKueue environment.
+
+For more details about the MultiKueue check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
+
+## Running tasks in MultiKueue environment
+
+To utilize this feature, you simply submit the job to the Manager cluster, targeting a ClusterQueue that has been configured for MultiKueue operation.
+
+Once submitted, Kueue automatically handles the delegation of the job to the appropriate worker clusters, requiring no additional configuration changes on your part.
```

Comment:
```suggestion
This page explains how to run tasks in MultiKueue environment for multi-cluster Job dispatching.

In particular, we focus on details of running your tasks using various Job CRDs.
```
TBH I think this is enough here as a starter.
Also currently, "To utilize this feature, you simply submit the job to the Manager cluster, targeting a ClusterQueue that has been configured for MultiKueue operation." is not that simple. details differ. For some CRDs we have managedBy defaulting, for some not, etc.